.github/workflows/deploy-enterprise.yml

```diff
@@ -65,6 +65,7 @@ jobs:
               job_spec/ARIA_AUTORIFT.yml
               job_spec/ARIA_RAIDER.yml
               job_spec/INSAR_ISCE.yml
+              job_spec/OPERA_DIST_S1.yml
```

Comment:
@cmarshak do you want this in `hyp3-a19-jpl` too? Or are you using tibet and nisar for different funding streams than a19? 

In that same vein, this is only a production version of the job (using the `latest` docker tag); do you want a test version as well? I'd recommend one if you're doing development. That could be accomplished by adding it to `hyp3-a19-jpl-test` or standing up a tibet and/or nisar test stack in their respective accounts (not too much work).